src/app/home/home.component.html

```diff
@@ -33,7 +33,9 @@
           <a href="/data-catalogue">
             <span class="home-text010"><span>Data Tracker</span></span>
           </a>
-          <span class="home-text012"><span>Gene list</span></span>
+          <a href="https://morphic-genelist-app.shinyapps.io/shiny_genesetexplore/" target="_blank">

```

Comment:
I believe external urls should be put in the environment file, to facilitate multiple environments.